### ISSUE TITLE
Type Parameters' angle bracket breaks in Docusaurus 2 (MDX)

### ISSUE THREAD

**Issue by MarcoPolo** [tgreyuk/typedoc-plugin-markdown#119]
Docusaurus 2 uses MDX to parse all markdown (even if it's not .mdx). It gets tripped up when there are type parameters in the code. i.e.:

```md
* Some type: Foo<T>
```

Will result in an error about no matching closing tag for `<T>`.

Maybe the docusaurus2 output can use the pointing angles instead? i.e. : ‹ and ›

**Comment by tgreyuk** [comment]
Implemented pointing angles (available in 2.3.0)

**Comment by darkbasic** [comment]
This isn't fixed at all for me:
```
### `Const` EmailToken

• **EmailToken**: *InjectionToken‹[Email](classes/email.md)‹any››* = new InjectionToken<Email<any>>('EmailToken')

*Defined in [types/Email.symbol.ts:4](https://github.com/darkbasic/accounts/blob/66cc15eb/packages/database-mikro-orm/src/types/Email.symbol.ts#L4)*

___

### `Const` ServiceToken

• **ServiceToken**: *InjectionToken‹[Service](classes/service.md)‹any››* = new InjectionToken<Service<any>>('ServiceToken')

*Defined in [types/Service.symbol.ts:4](https://github.com/darkbasic/accounts/blob/66cc15eb/packages/database-mikro-orm/src/types/Service.symbol.ts#L4)*

___

### `Const` SessionToken

• **SessionToken**: *InjectionToken‹[Session](classes/session.md)‹any››* = new InjectionToken<Session<any>>('SessionToken')

*Defined in [types/Session.symbol.ts:4](https://github.com/darkbasic/accounts/blob/66cc15eb/packages/database-mikro-orm/src/types/Session.symbol.ts#L4)*

___

### `Const` UserToken

• **UserToken**: *InjectionToken‹[IUser](interfaces/iuser.md)‹any, any, any››* = new InjectionToken<IUser<any, any, any>>('UserToken')

*Defined in [types/User.symbol.ts:4](https://github.com/darkbasic/accounts/blob/66cc15eb/packages/database-mikro-orm/src/types/User.symbol.ts#L4)*

## Functions

### `Const` getEmailCtor

▸ **getEmailCtor**‹**CustomUser**›(`__namedParameters`: object): *[EmailCtor](globals.md#emailctor)‹CustomUser›*

*Defined in [entity/Email.ts:34](https://github.com/darkbasic/accounts/blob/66cc15eb/packages/database-mikro-orm/src/entity/Email.ts#L34)*
```

Some tags use pointing angles, but not all of them! typedoc-plugin-markdown 2.4.2.
